VTEX - Marketplace APIs - Suggestions.json

```diff
@@ -2444,130 +2456,215 @@
+                            "Product": {
+                                "type": "array",
+                                "description": "Product's name.",
+                                "items": {
+                                    "type": "string",
+                                    "description": "Product name."
+                                }
+                            }
+                        }
+                    },
+                    "SpecificationsMapping": {
+                        "type": "array",
+                        "description": "This attribute maps product and SKU's specifications between the marketplace and the seller.",
+                        "items": {
+                            "type": "object",
+                            "description": "Object representing a single specification mapping between the marketplace and the seller.",
+                            "properties": {
+                                "SellerId": {
+                                    "type": "string",
+                                    "description": "The Seller ID."
+                                },
+                                "Mapping": {
+                                    "type": "object",
+                                    "description": "The attributes and values mapped between the marketplace and the seller."
+                                }
+                            }
+                        }
+                    },
+                    "MatchFlux": {
+                        "type": "string",
+                        "description": "Type of approval configuration that apply to received SKUs sent by sellers. The possible values are: \n\n`default`: the matcher approves the SKU. \n\n`manual`: manual SKU's approvals. \n\n`AutoApprove`: automatic SKU's approvals."
+                    }
                 }
             }
         },
-        "tags": [
-            {
-                "name": "Get Suggestions",
-                "description": "Search and filter all suggestions using specific criteria."
-            },
-            {
-                "name": "Manage Suggestions",
-                "description": "Send or delete SKU suggestions from the seller to marketplace."
-            },
-            {
-                "name": "Get Versions",
-                "description": "Search and filter all versions of suggestions, using specific criteria."
-            },
-            {
-                "name": "SKU Approval Settings"
+        "securitySchemes": {
+            "appKey": {
+                "type": "apiKey",
+                "in": "header",
+                "name": "X-VTEX-API-AppKey"
             },
-            {
-                "name": "Match Received SKUs",
-                "description": "Match SKU suggestions received in the marketplace."
+            "appToken": {
+                "type": "apiKey",
+                "in": "header",
+                "name": "X-VTEX-API-AppToken"
```

Comment:
Review security schemes as in template, including `appKey` and `appToken` descriptions and `VtexIdclientAutCookie`.